### ISSUE TITLE
sph_tributary xcent,ycent,zcent

### ISSUE THREAD

**Issue by dnarayanan** [dnarayanan/powderday#47]
should xcent,ycent,zcent in sph_tributary just come from cfg.par instead of the cell calculation as done?  what's the difference in the two?

**Comment by dnarayanan** [comment]
looks like they're exactly the same. this can be an easy fix once the arepo branch is merged into master

**Comment by dnarayanan** [comment]
looks like they're exactly the same.  once the arepo branch is merged in, we can just clean this up easily with something like:
```
x_cent  = ds.quan(cfg.model.x_cent,'code_length')
x_cent = x_cent.in_units('cm')
```

**Comment by dnarayanan** [comment]
addressed in ef6f472fafeeff15f128f345f01a9e905ac2ee05
